packages/react-components/src/atoms/__tests__/Dropdown.test.tsx

```diff
@@ -124,3 +124,20 @@ it('gets a green border when focused', () => {
     )?.borderColor.replace(/ /g, ''),
   ).toBe(fern.rgb.replace(/ /g, ''));
 });
+
+it('shows the dropdown grey out', () => {
```

Comment:
please do it as a "differential" test with a rerender (see e.g. `TextField.test`) to be sure you're actually testing styles that aren't just always there. One of the styles (and toBeDis/Enabled) would suffice imo